docs/usage/configuration/config_documentation.md

```diff
@@ -1764,6 +1764,28 @@ Example configuration:
 max_image_pixels: 35M
 ```
 ---
+### `prevent_downloads_from`
```

Comment:
The media options are all top-level, so I think we should probably put `media` somewhere in here?

```suggestion
### `prevent_media_downloads_from`
```

Or maybe:

```suggestion
### `media_domain_deny_list`
```

Reply:
(I'm very worried that `media_domain_deny_list` or similar communicates the wrong intent: one of the first questions I had on the MMR implementation was whether the config option stopped *other* domains from downloading *local* media)